rotkehlchen/api/rest.py

```diff
@@ -4359,7 +4359,7 @@ def refresh_general_cache(self) -> dict[str, Any]:
         caches = (
             ('curve pools', CacheType.CURVE_LP_TOKENS, query_curve_data, save_curve_data_to_cache, eth_node_inquirer),  # noqa: E501
             ('convex pools', CacheType.CONVEX_POOL_ADDRESS, query_convex_data, save_convex_data_to_cache, eth_node_inquirer),  # noqa: E501
-            ('velodrome pools', CacheType.VELODROME_POOL_ADDRESS, query_velodrome_data, save_velodrome_data_to_cache, optimism_inquirer),  # noqa: E501
+            ('velodrome pools', CacheType.VELODROME_POOL_ADDRESS, query_velodrome_like_data, save_velodrome_data_to_cache, optimism_inquirer),  # noqa: E501
```

Comment:
As asked elsewhere shouldn't we have one per chain?

Reply:
I'm not sure what you mean here. I was missing aerodrome in here if this is what you mean